lib/northbound.h

```diff
@@ -628,6 +628,13 @@ struct frr_yang_module_info {
 	 */
 	bool ignore_cfg_cbs;
 
+	/*
+	 * The NULL-terminated list of supported features.
+	 * Use ["*", NULL] to enable all features.
+	 * Use NULL to disable all features.
+	 */
+	const char **features;
```

Comment:
It is not clear to me what features are we referencing in this context and what purpose it serves.

Reply:
It's about about YANG model features: https://datatracker.ietf.org/doc/html/rfc7950#section-5.6.2. As we're working here with YANG modules I thought it was obvious, but I can specifically mention this in the comment I suppose.